### DISCUSSION TITLE
Wire gauges

### DISCUSSION THREAD

**Discussion by Astrolava** [bassamanator/everything-sovol-sv06#38] [Q&A]
Recently, the thermistor for my bed stopped reporting correct temperatures, and sometimes went to the deep negatives.
I'm sure one of the wires snapped inside, but at 50 dollars per spool, I really don't want to make any mistakes with the wire gauges. 
I got some guesses for what the gauges could be, but I don't want to risk buying the wrong spool of wire.
I figured while I'm replacing the thermistor wire, I could also extend some of the other wires to put my MCU outside my enclosure.
So, if anyone knows what gauges of wire, or possibly a source where to get them, please let me know.

**Comment by Heiko910** [comment]
I just did the same, one of the yellow wires (thermistor) was broken. Those
wires don't carry any significant current. I had some 20gauge in yellow
from the auto part store. I cut it off near the bed and near the connector
and solder in a new piece. Works great. I secured it to the rest of the
wires with electrical tape.

**Comment by Heiko910** [comment]
The solder connections I put some shrink tubing over it.

**Comment by bassamanator** [comment]
Unsure as to the exact size.

I really hope you don't have to spend that much, should cost $7-8 at your local hardware store for a few meters. Aliexpress will have plenty at a good price, but you have to wait a lot.
